msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -208,6 +208,15 @@ The app manifest describes how the app integrates into the Microsoft Teams platf
         "id": "AAD App ID",
         "resource": "Resource URL for acquiring auth token for SSO"
     },
+    "graphConnector": "https://contoso.com/connector/notification",
+    "showLoadingIndicator": false,
+    "isFullScreen": false,
+    "defaultBlockUntilAdminAction": false,
+    "publisherDocsUrl": "https://contoso.com/teamtabapp/admin-doc",
+    "scopeConstraints": {
```

Comment:
@erikadoyle I could find no example snippet or use cases of scopeConstraints in the samples folder or in the docs, any guidance on this portion?

Reply:
Just messaged you with ideas to look for more info--